pkg/cmd/template/schema_author_test.go

```diff
@@ -756,6 +756,198 @@ schema.yml:
 				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
 			})
 
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+	})
+	t.Run("when schema/examples annotation value", func(t *testing.T) {
+		t.Run("is empty", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples
+  4 | key: val
+    |
+
+    = found: missing value in @schema/examples (by schema.yml:3)
+    = expected: tuple with description and example
+`
+
+			filesToProcess := files.NewSortedFiles([]*files.File{
+				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
+			})
+
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+		t.Run("is not a tuple", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/examples "example value"
+key: val
+`
+			expectedErr := `
+Invalid schema
+==============
+
+syntax error in @schema/examples annotation
+schema.yml:
+    |
+  3 | #@schema/examples "example value"
+  4 | key: val
+    |
+
+    = found: non tuple in @schema/examples (by schema.yml:3)
```

Comment:
Do we have access to the type of the actual argument? We'll learn in the expectation that "tuple" is what we want. 

```suggestion
    = found: string for @schema/examples (at schema.yml:3)
```

nit: s/by/at/ ??